README.md

```diff
@@ -140,6 +140,18 @@ JSON.stringify(img._corners)
 => "[{"lat":51.52,"lng":-0.1},{"lat":51.52,"lng":-0.14},{"lat":51.5,"lng":-0.1},{"lat":51.5,"lng":-0.14}]"
 ```
 
+### Adding "DOMStrings" for custom toolbars
+
+Custom "DOMStrings" that define the custom toolbars can easily be added by describing those in the `src/edit/tools/DistortableImage.Guides.js` file and referencing those in the `addToolbar` method inside the `src/edit/DistortableImage.Edit.js` file.
```

Comment:
Are "domstrings" html snippets? if so, perhaps we could just add `(i.e. strings of HTML)` Thanks!!!